win32ss/gdi/ntgdi/cliprgn.c

```diff
@@ -776,16 +778,16 @@ CLIPPING_UpdateGCRegion(PDC pDC)
         pDC->prgnAPI = NULL;
     }
 
-    if (pDC->prgnRao)
-        REGION_Delete(pDC->prgnRao);
-
-    pDC->prgnRao = IntSysCreateRectpRgn(0,0,0,0);
-
-    ASSERT(pDC->prgnRao);
-
     if (pDC->dclevel.prgnMeta || pDC->dclevel.prgnClip)
     {
         pDC->prgnAPI = IntSysCreateRectpRgn(0,0,0,0);
+        if (!pDC->prgnAPI)
+        {
+            /* Best we can do here. Better than crashing. */
+            ERR("Failed to allocate prgnAPI! Expect drawing issues!\n");
```

Comment:
```suggestion
            ERR("Failed to allocate prgnAPI! Expect drawing issues\n");
```

Reply:
`ERR("Failed to provide useful feedback! Stop wasting my time!!!!\n");`